rules/.gherkin-lintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "no-files-without-scenarios" : "on",
+  "no-unnamed-features": "on",
+  "no-unnamed-scenarios": "on",
+  "no-dupe-scenario-names": ["on", "in-feature"],
+  "no-dupe-feature-names": "on",
+  "no-partially-commented-tag-lines": "on",
+  "indentation": "off",
```

Comment:
this should disable the indentation rules, the rest is jsut the default from https://github.com/github/super-linter/blob/master/TEMPLATES/.gherkin-lintrc